Jenkinsfile

```diff
@@ -68,7 +68,8 @@ pipeline
             sh '''
                cd offline
                pdflatex -interaction=nonstopmode publication_list.tex
-               biber publication_list
+               biber publication_list | tee biber.log
+               grep WARN biber.log | grep Duplicate && exit 1
```

Comment:
```suggestion
               biber publication_list
               grep WARN publication_list.blg | grep Duplicate && exit 1
```

`biber` already writes a logfile, which you could use alternatively. I leave the choice to you :)

```
$ grep WARN publication_list.blg | grep Duplicate
[18283] Biber.pm:130> WARN - Duplicate entry key: 'Jodlbauer2018' in file '../publications-2018.bib', skipping ...
[18283] Biber.pm:130> WARN - Duplicate entry key: 'Guo2018' in file '../publications-2018.bib', skipping ...
[18283] Biber.pm:130> WARN - Duplicate entry key: 'Deolmi2018' in file '../publications-2018.bib', skipping ...
[18283] Biber.pm:130> WARN - Duplicate entry key: 'Cheng2018' in file '../publications-2018.bib', skipping ...
[18283] Biber.pm:130> WARN - Duplicate entry key: 'Cangiani2018' in file '../publications-2018.bib', skipping ...
[18283] Biber.pm:130> WARN - Duplicate entry key: 'Zhang2018' in file '../publications-2018.bib', skipping ...
[18284] Biber.pm:130> WARN - Duplicate entry key: 'Carreo2018' in file '../publications-2018.bib', skipping ...
[18284] Biber.pm:130> WARN - Duplicate entry key: 'Wang2018' in file '../publications-2018.bib', skipping ...
[18285] Biber.pm:130> WARN - Duplicate entry key: 'Arbogast2019' in file '../publications-2019.bib', skipping ...
[18285] Biber.pm:130> WARN - Duplicate entry key: 'Guo2019' in file '../publications-2019.bib', skipping ...
[18286] Biber.pm:130> WARN - Duplicate entry key: 'Hagstrom2019' in file '../publications-2019.bib', skipping ...
[18286] Biber.pm:130> WARN - Duplicate entry key: 'Balicki2019' in file '../publications-2019.bib', skipping ...
[18286] Biber.pm:130> WARN - Duplicate entry key: 'EbnaHai2019' in file '../publications-2019.bib', skipping ...
[18286] Biber.pm:130> WARN - Duplicate entry key: 'Bzowski2019' in file '../publications-2019.bib', skipping ...
[18286] Biber.pm:130> WARN - Duplicate entry key: 'Freddi2019' in file '../publications-2019.bib', skipping ...
[18286] Biber.pm:130> WARN - Duplicate entry key: 'Alarydah2019' in file '../publications-2019.bib', skipping ...
[18286] Biber.pm:130> WARN - Duplicate entry key: 'Carreo2019' in file '../publications-2019.bib', skipping ...
[18286] Biber.pm:130> WARN - Duplicate entry key: 'Vassaux2019' in file '../publications-2019.bib', skipping ...
[18286] Biber.pm:130> WARN - Duplicate entry key: 'Dang2019' in file '../publications-2019.bib', skipping ...
[18286] Biber.pm:130> WARN - Duplicate entry key: 'Heltai2019' in file '../publications-2019.bib', skipping ...
[18286] Biber.pm:130> WARN - Duplicate entry key: 'Kim2019' in file '../publications-2019.bib', skipping ...
[18287] Biber.pm:130> WARN - Duplicate entry key: 'Bonito2019' in file '../publications-2019.bib', skipping ...
[18287] Biber.pm:130> WARN - Duplicate entry key: 'Carreo2019' in file '../publications-2019.bib', skipping ...
[18287] Biber.pm:130> WARN - Duplicate entry key: 'Vassaux2019' in file '../publications-2019.bib', skipping ...
[18287] Biber.pm:130> WARN - Duplicate entry key: 'Konschin2019' in file '../publications-2019.bib', skipping ...
[18287] Biber.pm:130> WARN - Duplicate entry key: 'Robey2019' in file '../publications-2019.bib', skipping ...
[18287] Biber.pm:130> WARN - Duplicate entry key: 'Das2019' in file '../publications-2019.bib', skipping ...
[18287] Biber.pm:130> WARN - Duplicate entry key: 'Zhang2019' in file '../publications-2019.bib', skipping ...
```